integration-tests/testing-framework/src/test/java/io/quarkiverse/githubapp/it/testingframework/TestingFrameworkTest.java

```diff
@@ -222,37 +222,38 @@ void clientProvider() {
                             installation2);
                     Mockito.when(app.listInstallations()).thenReturn(appInstallations);
 
-                    Mockito.when(installation1Repo1.getName()).thenReturn("quarkus");
+                    Mockito.when(installation1Repo1.getFullName()).thenReturn("quarkusio/quarkus");
                     PagedSearchIterable<GHRepository> installation1Repos = MockHelper.mockPagedIterable(installation1Repo1);
                     Mockito.when(installation1.listRepositories())
                             .thenReturn(installation1Repos);
 
-                    Mockito.when(installation2Repo1.getName()).thenReturn("quarkus-github-app");
-                    Mockito.when(installation2Repo2.getName()).thenReturn("quarkus-github-api");
+                    Mockito.when(installation2Repo1.getFullName()).thenReturn("quarkiverse/quarkus-github-app");
+                    Mockito.when(installation2Repo2.getFullName()).thenReturn("quarkiverse/quarkus-github-api");
```

Comment:
Yeah I stumbled upon that the other day and refrained from doing it as I thought it could break some projects. But if it's in your way, let's do it.

Reply:
Note the problem was merely in my test, not in the main code. So this does not break anything.